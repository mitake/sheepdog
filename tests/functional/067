#!/bin/bash

# Test concurrent start up

seq=`basename $0`
echo "QA output created by $seq"

here=`pwd`
tmp=/tmp/$$
status=1        # failure is the default!

# get standard environment, filters and checks
. ./common.rc
. ./common.filter

_cleanup

for i in 0 1 2; do
    _start_sheep $i
done

_wait_for_sheep 3

_cluster_format

$COLLIE node list

# make sure sheep1 and sheep2 quit simutaniously
_kill_sheep 1 &
_kill_sheep 2
_wait_for_sheep_stop 1
_wait_for_sheep_stop 2

sleep 3

for i in 2 1 ; do
    _start_sheep $i
done

_kill_sheep 0
_wait_for_sheep 2 2

sleep 1

echo check whether all nodes have the same cluster info
for i in 1 2; do
    $COLLIE node list -p 700$i
done
