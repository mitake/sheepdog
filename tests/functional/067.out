QA output created by 067
using backend plain store
  Id   Host:Port         V-Nodes       Zone
   0   127.0.0.1:7000      	64          0
   1   127.0.0.1:7001      	64          1
   2   127.0.0.1:7002      	64          2
check whether all nodes have the same cluster info
  Id   Host:Port         V-Nodes       Zone
   0   127.0.0.1:7001      	64          1
   1   127.0.0.1:7002      	64          2
  Id   Host:Port         V-Nodes       Zone
   0   127.0.0.1:7001      	64          1
   1   127.0.0.1:7002      	64          2
